jupyter_kernel_mgmt/subproc/manager.py

```diff
@@ -123,13 +124,14 @@ def signal(self, signum):
         if hasattr(os, "getpgid") and hasattr(os, "killpg"):
             try:
                 pgid = os.getpgid(self.kernel.pid)
-                os.killpg(pgid, signum)
+                await maybe_future(os.killpg(pgid, signum))
```

Comment:
Ditto here - os.killpg is a sync function, as far as I know, so there's no need to wrap it in `maybe_future`.

Reply:
I ditto your ditto. :smile: